src/panel/layouts.js

```diff
@@ -26,13 +21,20 @@ export function getMapPaddings({ isMobile, isDirectionsActive, isIframe }) {
   const topUIElement = isDirectionsActive ? '.direction-panel' : '.top_bar';
   const topUIHeight = document.querySelector(topUIElement)?.clientHeight || 0;
   return {
-    bottom: mobileBottomPanelHeight + ADDITIONAL_PADDING / 2,
+    bottom: MOBILE_BOTTOM_PADDING,
```

Comment:
Are we sure that there is no situation where a dynamic value would still be required here to fit the remaining space at the top of a larger panel ?

Reply:
I haven't seen any for now. On the contrary we already display things below the panel (category POIs for example), and when we resize the panel manually we don't want that to have any effect on the map center, it slides above it.
But I agree this isn't ideal like that.